feature-libs/user/profile/components/update-password/update-password-component.service.ts

```diff
@@ -93,7 +94,16 @@ export class UpdatePasswordComponentService {
     });
   }
 
-  protected onError(_error: Error): void {
+  protected onError(_error: HttpErrorModel | Error): void {
+    if (
+      _error instanceof HttpErrorModel &&
+      _error.details?.[0].type === 'AccessDeniedError'
```

Comment:
@Zeyber , thanks for reminding. For other error scenario, it has been handled by the core http-interceptors. Only  special scenario of agent modify customer's password is the missing one. And for other negative scenarios, I have tested them manually, it works well.

Reply:
A notation with the underscore `_error` suggests this parameter is unused. Since you added some logic, the underscore should be removed.

Apart from that, changing the name of the parameter as well as its type may be considered a breaking change which we want to avoid in minor releases. Please double-check whether such a change may cause some issues if e.g. customers have some custom implementation of this method in their application (it is possible because this method has a `protected` access modifier)

Reply:
@pawelfras, If customer have some custom implementation with override current file method, the ui issue will still be there, this is the current impact that it might be.

Reply:
@evy-yang01 @Zeyber After rethinking the problem, union type you proposed won't cause breaking changes because this union contained previous type (and method behaves as previous if error type === `Error`). The feature flag is not needed. Sorry for the confusion.